src/cabal-details.js

```diff
@@ -671,6 +673,26 @@ class CabalDetails extends EventEmitter {
 
     // Load moderation state
     const loadModerationState = (cb) => {
+	  const promises = [this.moderation.getAdmins(), this.moderation.getMods()]
+	  // get all moderation actions issued by our current mods & admins
+	  Promise.all(promises).then(results => {
+		const keys = results[0].concat(results[1])
+		keys.forEach(key => {
+		  const write = (row, enc, next) => {
+			if (!row) return
+			const name = this.users[key].name || key.slice(0, 8)
+			const target = this.users[row.content.id].name || row.content.id.slice(0, 8)
+			const action = row.type.split("/")[1]
+			const reason = row.content.reason
+			const role = row.content.flags[0]
+			const text = `${name} ${action === "remove" ? "removed" : "added"} ${target} as ${role} ${reason ? "reason: " + reason : ''}`
+			this.addStatusMessage({ text, timestamp: row.timestamp })
+			next()
+		  }
+		  const end = (next) => { next() }
+		  pump(this.core.moderation.listModerationBy(key), to.obj(write, end))
```

Comment:
i might just skip this logic fwiw; what it does is it prints out all messages that occured for all yr mods/admins on first startup of cabal